src/OpenAL/OpenToolkit.OpenAL/Native/ApiAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OpenToolkit.Core.Loader;
+
+namespace OpenToolkit.Audio.OpenAL
+{
+    public class ApiAttribute : Attribute
+    {
+        public string DllName { get; set; }
+
+        public Type PlatformNameContainerType { get; set; }
+
+        public ApiAttribute(string dllName, Type platformNameContainerType)
+        {
+            DllName = dllName;
```

Comment:
Maybe a `string.IsNotNullOrEmpty` check here as well since the dllName is mandatory